packages/tupaia-web/src/api/queries/useEntities.ts

```diff
@@ -34,29 +38,40 @@ export const useEntities = (
           ],
         },
         ...axiosConfig,
-      }),
+      });
+    },
     {
       enabled,
     },
   );
 };
 
-export const useEntitiesWithLocation = (projectCode?: string, entityCode?: string) =>
-  useEntities(projectCode, entityCode, {
-    params: {
-      includeRoot: true,
-      fields: [
-        'parent_code',
-        'code',
-        'name',
-        'type',
-        'bounds',
-        'region',
-        'point',
-        'location_type',
-        'image_url',
-        'attributes',
-        'child_codes',
-      ],
+export const useEntitiesWithLocation = (
+  projectCode?: string,
+  entityCode?: string,
+  axiosConfig?: AxiosRequestConfig,
+  queryOptions?: QueryObserverOptions,
+) =>
+  useEntities(
+    projectCode,
+    entityCode,
+    {
+      params: {
+        ...{ ...axiosConfig?.params },
+        fields: [
```

Comment:
This is nice. Can you think of any use case where we may want to overwrite those fields?

Reply:
No I don't think so and I kind of don't want to because the less variations of params there are the more caching there is